service/router.go

```diff
@@ -15,6 +16,14 @@ const (
 	errWalletLockMsg = "error locking wallet: "
 )
 
+func (s *ShamirCoordinatorService) AddToQueue(err error) bool {
+	// Invalid Bitcoin address response error
+	if strings.Contains(err.Error(), "Invalid Bitcoin address:") || !strings.HasSuffix(err.Error(), ": -5") {
```

Comment:
I think it needs to be `strings.Contains(err.Error(), "Invalid Bitcoin address:") || strings.HasSuffix(err.Error(), ": -5")`

Reply:
true! a copy and past error thx!